_data/guides.yaml

```diff
@@ -206,7 +206,7 @@ categories:
         url: /guides/container-image
         description: Learn how to build and push container images with Jib, S2I or Docker as part of the Quarkus build.
       - title: Deploying Quarkus Applications on Kubernetes
-        url: /guides/kubernetes
+        url: /guides/deploying-to-kubernetes
```

Comment:
both URL's responds....any particular reasons we got both pointing to the same ?

Reply:
Hum, looks like we have a duplicate because I fixed it elsewhere: https://github.com/quarkusio/quarkusio.github.io/commit/dae02801ab2ff456cd0496adfc1343bd0923e2d3#diff-601fe746a96531933be1159bf46dec51R212 :)

Reply:
@geoand are you sure you have latest develop.

This is it: https://github.com/quarkusio/quarkusio.github.io/blob/develop/_data/guides.yaml#L212 . And the website looks OK.

Reply:
Not sure TBH... I'll close this since it's been done